core/build.gradle

```
@@ -25,20 +25,20 @@ dependencies {
     // IDEs do not like it when 'core' references XSeries directly
     // but subprojects must reference XSeries as shaded.
     api 'com.github.cryptomorin:XSeries:9.4.0'
-    implementation 'org.bstats:bstats-bukkit:3.0.0'
+    implementation 'org.bstats:bstats-bukkit:3.0.2'
     compileOnly 'net.luckperms:api:5.4'
-    compileOnly "me.lokka30:treasury-api:1.2.1-SNAPSHOT"
+    compileOnly "me.lokka30:treasury-api:2.0.2-SNAPSHOT"
     compileOnly 'de.jeff_media:ChestSortAPI:13.0.0-SNAPSHOT'
-    compileOnly 'com.palmergames.bukkit.towny:towny:0.98.6.0'
-    implementation 'com.zaxxer:HikariCP:4.0.3'
-    implementation 'net.wesjd:anvilgui:1.6.6-SNAPSHOT'
+    compileOnly 'com.palmergames.bukkit.towny:towny:0.99.5.4'
+    implementation 'com.zaxxer:HikariCP:5.0.1'
```

```suggestion
    // Last version to support Java 8
    implementation 'com.zaxxer:HikariCP:4.0.3'
```
From the build output it seems this version of Hikari is needed